test/route.test.js

```diff
@@ -217,7 +217,7 @@ test('/v1/foobar/ should redirect to ./static/index.html - in plugin', async (t)
   t.plan(3)
   const fastify = Fastify()
 
-  fastify.register(async function (fastify, options) {
+  await fastify.register(async function (fastify, options) {
```

Comment:
This is not needed, inject() will register all plugins.

Reply:
Didnt know that .inject will finalize the encapsulation.

```suggestion
  fastify.register(async function (fastify, options) {
```

Reply:
Aw fudge, I didn't know this either. It's there in the documentation as well!

And here I am awaiting `ready` in my own repo tests prior to injecting.